receiver/signalfxreceiver/factory_test.go

```diff
@@ -39,14 +39,14 @@ func TestCreateReceiverMetricsFirst(t *testing.T) {
 	cfg.Endpoint = "localhost:1" // Endpoint is required, not going to be used here.
 
 	params := component.ReceiverCreateParams{Logger: zap.NewNop()}
-	mReceiver, err := factory.CreateMetricsReceiver(context.Background(), params, cfg, exportertest.NewNopMetricsExporter())
+	mReceiver, err := factory.CreateMetricsReceiver(context.Background(), params, cfg, consumertest.NewMetricsNop())
 	assert.Nil(t, err, "receiver creation failed")
 	assert.NotNil(t, mReceiver, "receiver creation failed")
 
 	_, err = factory.CreateTraceReceiver(context.Background(), component.ReceiverCreateParams{Logger: zap.NewNop()}, cfg, nil)
 	assert.Equal(t, err, configerror.ErrDataTypeIsNotSupported)
 
-	lReceiver, err := factory.CreateLogsReceiver(context.Background(), component.ReceiverCreateParams{Logger: zap.NewNop()}, cfg, new(exportertest.SinkLogsExporter))
+	lReceiver, err := factory.CreateLogsReceiver(context.Background(), component.ReceiverCreateParams{Logger: zap.NewNop()}, cfg, consumertest.NewLogsNop())
```

Comment:
Could you please double-check this change? It was a SinkExporter before, it's a Nop now. There are a couple of similar changes in this test.

Reply:
Yes :) my script got smarter. If you pass the new object to the factory and don't hold a reference, no need to store them because you cannot retrieve them :)

Reply:
Is there really a script behind this PR? O_O

Reply:
It is called sed :))